lab_2/main.py

```diff
@@ -170,11 +183,8 @@ def find_lcs(first_sentence_tokens: tuple, second_sentence_tokens: tuple, lcs_ma
                     lcs = tuple(lcs)
 
                     return lcs
-
                 return ()
-
         return ()
-
     return ()
```

Comment:
so many `return ()`, can you add one simple check at the beginning and return () there?